core/src/main/kotlin/javadoc/tags.kt

```diff
@@ -130,36 +153,37 @@ class ThrowsTagAdapter(val holder: Doc, val type: ClassDocumentationNodeAdapter,
     override fun exceptionName(): String = type.qualifiedTypeName()
 }
 
-class ReturnTagAdapter(val module: ModuleNodeAdapter, val holder: Doc, val content: List<ContentNode>) : Tag {
+class ReturnTagAdapter(val module: ModuleNodeAdapter, val holder: Doc, val content: List<ContentNode>,
+                       private val formatService: StructuredFormatService?) : Tag {
     override fun name(): String = "@return"
     override fun kind() = name()
     override fun holder() = holder
     override fun position(): SourcePosition? = holder.position()
 
     override fun text(): String = "@return $content" // Seems has no effect, so used for debug
-    override fun inlineTags(): Array<Tag> = buildInlineTags(module, holder, content).toTypedArray()
+    override fun inlineTags(): Array<Tag> = buildInlineTags(module, holder, content, formatService).toTypedArray()
     override fun firstSentenceTags(): Array<Tag> = inlineTags()
 }
 
-fun buildInlineTags(module: ModuleNodeAdapter, holder: Doc, tags: List<ContentNode>): List<Tag> = ArrayList<Tag>().apply { tags.forEach { buildInlineTags(module, holder, it, this) } }
+fun buildInlineTags(module: ModuleNodeAdapter, holder: Doc, tags: List<ContentNode>, formatService: StructuredFormatService?): List<Tag> = ArrayList<Tag>().apply { tags.forEach { buildInlineTags(module, holder, it, this, formatService) } }
```

Comment:
It possible to store formatService in ModuleNodeAdapter, then you won't need to bypass it all around.